lib/pos_snapshot_entries.go

```diff
@@ -43,6 +43,10 @@ func (snapshotGlobalParam SnapshotGlobalParam) ToString() string {
 	return string(snapshotGlobalParam)
 }
 
+func (bav *UtxoView) GetCurrentGlobalParam(field SnapshotGlobalParam) uint64 {
```

Comment:
This is used by backend when we update the GlobalParamsEntry. We only want to update if the passed in value differs from the current value. So we need a getter for the current value (and not just the snapshot value like we had before).